src-tool/Pact/Analyze/Types/Capability.hs

```diff
@@ -19,19 +19,27 @@ import           Pact.Analyze.PactSFunArray (eitherArray, mkPactSFunArray)
 import           Pact.Analyze.Types.Shared
 import           Pact.Analyze.Types.Types
 
+
+data CapabilityAnnot
```

Comment:
If we never use these values, why record them?

Reply:
My intention here was to increase readability. In this particular case, a flag indicating the cap is managed or an event would be enough.